docs/index.md

```diff
@@ -0,0 +1,196 @@
+
+#### Collection Index
+
+In this record, collections are mapped by their IDs.
+
+| **Length** (bytes) | `1`               | `64`                   |
+|:-------------------|:------------------|:-----------------------|
+| **Type**           | byte              | flow.Identifier        |
+| **Description**    | Index type prefix | Collection ID         |
+| **Example Value**  | `10`              | `45D66Q565F5DEDB[...]` |
+
+The value stored at that key is the **CBOR-encoded [flow.LightCollection](https://pkg.go.dev/github.com/onflow/model/flow#LightCollection)** with the referenced ID.
+
+#### Collection Guarantee Index
+
+In this record, collections guarantees are mapped by their collection IDs.
+
+| **Length** (bytes) | `1`               | `64`                   |
+|:-------------------|:------------------|:-----------------------|
+| **Type**           | byte              | flow.Identifier        |
+| **Description**    | Index type prefix | Collection ID         |
+| **Example Value**  | `10`              | `45D66Q565F5DEDB[...]` |
+
+The value stored at that key is the **CBOR-encoded [flow.CollectionGuarantee](https://pkg.go.dev/github.com/onflow/model/flow#CollectionGuarantee)** with the referenced ID.
+
+#### Block Collection Index
+
+In this index, block IDs are mapped to the IDs of the collections within their block.
+
+| **Length** (bytes) | `1`               | `64`                   |
+|:-------------------|:------------------|:-----------------------|
+| **Type**           | byte              | flow.Identifier        |
+| **Description**    | Index type prefix | Block ID               |
+| **Example Value**  | `11`              | `45D66Q565F5DEDB[...]` |
+
+The value stored at that key is the **CBOR-encoded slice of [flow.Identifier](https://pkg.go.dev/github.com/onflow/model/flow#Identifier)** for the collections within the referenced block.
+
+#### Transaction Result Index
+
+In this index, transaction IDs are mapped to their results.
+
+| **Length** (bytes) | `1`               | `64`                   |
+|:-------------------|:------------------|:-----------------------|
+| **Type**           | byte              | flow.Identifier        |
+| **Description**    | Index type prefix | Transaction ID         |
+| **Example Value**  | `12`              | `45D66Q565F5DEDB[...]` |
+
+The value stored at that key is the **CBOR-encoded [flow.TransactionResult](https://pkg.go.dev/github.com/onflow/model/flow#TransactionResult)** for the referenced transaction.
+
+#### Seals Index
+
+In this index, seals are mapped by their IDs.
+
+| **Length** (bytes) | `1`               | `64`                   |
+|:-------------------|:------------------|:-----------------------|
+| **Type**           | byte              | flow.Identifier        |
+| **Description**    | Index type prefix | Seal ID                |
+| **Example Value**  | `14`              | `45D66Q565F5DEDB[...]` |
+
```

Comment:
The value stored at that key is the **CBOR-encoded [flow.Seal](https://pkg.go.dev/github.com/onflow/flow-go/model/flow#Seal)** with the referenced ID.